README.md

```diff
@@ -71,83 +71,148 @@ usage: java -jar vcf-inheritance-matcher.jar -i <arg> [-o <arg>] [-pd
                              (.ped).
  -pb,--probands <arg>        Comma-separated list of proband individual
                              identifiers.
+ -c,--classes     			 Values in the VIPC subfield of VEP for which
+ 							 inheritance should be calculated.
  -f,--force                  Override the output file if it already
                              exists.
  -d,--debug                  Enable debug mode (additional logging).
 ```
 
-### Subinheritance modes:
+## Inheritance patterns
+- AR: Autosomal recessive
+- AD: Autosomal dominant
 - XLR: X-linked recessive
 - XLD: X-linked dominant
+- YL: Y-linked
+- MT: Mitochondrial
 - AR_C: Autosomal recessive compound hetrozygote
-- AD_IP: Autosomal dominant incomplete penetrance. 
-
-### Inheritance mode rules
-Possible inheritance modes are calculated on the following rules:
-#### AR:
-- Affected samples have to be homozygote ALT.
-- Unaffected samples cannot be homozygous ALT.
-#### AR compound hetrozygote:
-##### For unphased data:
-- Affected samples need to have both variants.
-- Unaffected samples cannot have both variants.
-##### For phased data:
-- Affected samples need to have both variants on different alleles.
-- Unaffected samples cannot have both variants on different alleles, however they can have both variants on the same alleles..
-#### AD:
-- Affected samples have to carry the ALT allele.
-Unaffected samples have to be homozygous REF.
-#### AD imcomplete penetrance:
-- Affected samples have to carry the ALT allele.
-- Unaffected samples have to be homozygous REF, unless the gene on which the variant lies is also on the provided non-penetrance list.
-#### XLD:
-- Affected samples have to have at least one ALT allele.
-- Male unaffected patients cannot have the ALT allele, female unaffected samples can have a single ALT allele due to X inactivation.
-#### XLR:
-- Female affected samples have to be homozygous ALT, male affected patients have to be homozygous ALT or have only the ALT allele.
-- Female unaffected samples cannot be homozygous ALT, males cannot be homozygous ALT and connot have only the REF allele.
-#### XL:
-- If the variant is XLD or XLR it is also considered XL.
-#### Denovo:
-##### On regular chromosomes:
-- Variant are considered denovo if one of the ALT alleles of the proband is not inherited from a parent.
-##### On the X chromosome: 
-- For male probands variants are considered denovo if mother does not have the ALT allele.
-- For female probands variants are considered denovo following the same rules as for the other chromosomes.
-
-### Running without pedigree file
+- AD_IP: Autosomal dominant incomplete penetrance
+
+## Inheritance pattern rules
+### General rules
+For inheritance matching all the members in a family are considered.
```

Comment:
this tool works for contigs such as `chr11_KI270721v1_random` and `chrY_KI270740v1_random`?

Reply:
I added a comment about this a few lines down in the README